ivadomed/main.py

```diff
@@ -214,12 +214,20 @@ def cmd_train(context):
         for param in model.parameters():
             param.requires_grad = False
 
+        # TMP
+        model.decoder = models.Decoder(out_channel=context['out_channel'],
```

Comment:
https://github.com/neuropoly/ivado-medical-imaging/issues/189

Reply:
So for now it retrains the entire decoder?